apps/service_define/config/dev.exs

```diff
@@ -14,9 +29,7 @@ config :service_define, DefineWeb.Endpoint,
       "../priv/static/",
       cd: Path.expand("../assets", __DIR__)
     ]
-  ]
-
-config :service_define, DefineWeb.Endpoint,
+  ],
   live_reload: [
```

Comment:
Did you mean to delete this line that now puts live reload under the main app and not under the endpoint?

Reply:
`live_reload` is still under the endpoint.  This diff is just really confusing I think.

Reply:
sure is, thanks lol